### PR TITLE
Add js examples

### PR DESCRIPTION
**Pull request by el8rbawy**

### CODE REVIEW

**Review by JajarGaming — APPROVED** [review]

**Review by el8rbawy — COMMENTED** [review]
DONE

**Review by JajarGaming — APPROVED** [review]
LGTM

**Comment by JajarGaming** [comment]
Just wait... Stuff doesn't get pushed quick

File: content/docs/resources/mapmanager/events/onClientGameTypeStart.md [hunk 1]
```diff
@@ -14,4 +14,9 @@ string resourceName
 Examples
 --------
 
-TODO
+##### JavaScript Example:
+```js
+on('onClientMapStart', name => {
```
Comment (AvarianKnight) [inline content/docs/resources/mapmanager/events/onClientGameTypeStart.md:19]:
```suggestion
on('onClientGameTypeStart', (name) => {
```

**Pull request merged**